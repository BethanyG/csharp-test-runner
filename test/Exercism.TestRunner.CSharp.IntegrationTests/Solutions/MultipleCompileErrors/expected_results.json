{
  "status": "error",
  "message": "FakeTest.cs(5,66): error CS0117: 'Fake' does not contain a definition for 'Add'\nFakeTest.cs(7,66): error CS0117: 'Fake' does not contain a definition for 'Sub'",
  "tests": []
}