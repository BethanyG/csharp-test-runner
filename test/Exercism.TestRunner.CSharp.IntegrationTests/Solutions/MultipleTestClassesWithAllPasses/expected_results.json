{
  "status": "pass",
  "tests": [
    {
      "name": "FakeTests.Add_should_add_numbers",
      "status": "pass"
    },
    {
      "name": "FakeTests.Sub_should_subtract_numbers",
      "status": "pass"
    },
    {
      "name": "FakeTests.Mul_should_multiply_numbers",
      "status": "pass"
    },
    {
      "name": "FooTests.Upper_should_uppercase_string",
      "status": "pass"
    },
    {
      "name": "FooTests.Lower_should_lowercase_string",
      "status": "pass"
    }
  ]
}