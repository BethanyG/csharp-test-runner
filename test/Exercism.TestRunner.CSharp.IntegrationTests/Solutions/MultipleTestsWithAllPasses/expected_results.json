{
  "status": "pass",
  "tests": [
    {
      "name": "Add should add numbers",
      "test": "FakeTest.Add_should_add_numbers",
      "expected": "2",
      "status": "pass"
    },
    {
      "name": "Sub should subtract numbers",
      "test": "FakeTest.Sub_should_subtract_numbers",
      "expected": "4",
      "status": "pass"
    },
    {
      "name": "Mul should multiply numbers",
      "test": "FakeTest.Mul_should_multiply_numbers",
      "expected": "6",
      "status": "pass"
    }
  ]
}