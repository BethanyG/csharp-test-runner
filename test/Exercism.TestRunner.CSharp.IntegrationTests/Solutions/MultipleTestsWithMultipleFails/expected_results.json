{
  "status": "fail",
  "tests": [
    {
      "name": "Add should add numbers",
      "test": "FakeTest.Add_should_add_numbers",
      "status": "fail",
      "message": "Assert.Equal() Failure\nExpected: 3\nActual:   2"
    },
    {
      "name": "Sub should subtract numbers",
      "test": "FakeTest.Sub_should_subtract_numbers",
      "status": "pass"
    },
    {
      "name": "Mul should multiply numbers",
      "test": "FakeTest.Mul_should_multiply_numbers",
      "status": "fail",
      "message": "Assert.Equal() Failure\nExpected: 7\nActual:   6"
    }
  ]
}