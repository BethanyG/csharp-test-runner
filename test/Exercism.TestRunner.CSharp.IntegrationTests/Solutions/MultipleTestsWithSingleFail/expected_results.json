{
  "status": "fail",
  "tests": [
    {
      "name": "Add should add numbers",
      "test": "FakeTest.Add_should_add_numbers",
      "status": "pass"
    },
    {
      "name": "Sub should subtract numbers",
      "test": "FakeTest.Sub_should_subtract_numbers",
      "status": "pass"
    },
    {
      "name": "Mul should multiply numbers",
      "test": "FakeTest.Mul_should_multiply_numbers",
      "status": "fail",
      "message": "Assert.Equal() Failure\nExpected: 5\nActual:   6"
    }
  ]
}