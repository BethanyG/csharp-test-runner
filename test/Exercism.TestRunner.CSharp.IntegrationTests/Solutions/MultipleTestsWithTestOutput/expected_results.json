{
  "status": "fail",
  "tests": [
    {
      "name": "Add should add numbers",
      "test": "FakeTest.Add_should_add_numbers",
      "status": "pass",
      "output": "String without params output"
    },
    {
      "name": "Sub should subtract numbers",
      "test": "FakeTest.Sub_should_subtract_numbers",
      "status": "pass"
    },
    {
      "name": "Mul should multiply numbers",
      "test": "FakeTest.Mul_should_multiply_numbers",
      "status": "fail",
      "message": "Assert.Equal() Failure\nExpected: 7\nActual:   6",
      "output": "String with params output\nValues used:\n2, True\n-----"
    }
  ]
}