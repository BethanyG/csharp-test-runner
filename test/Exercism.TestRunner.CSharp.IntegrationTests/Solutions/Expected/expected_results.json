{
  "status": "error",
  "message": "FakeTest.cs:75: Member \u0027Fake.Add(int, int)\u0027 cannot be accessed with an instance reference; qualify it with a type name instead\nFakeTest.cs:83: Member \u0027Fake.Add(int, int)\u0027 cannot be accessed with an instance reference; qualify it with a type name instead\nFakeTest.cs:91: Member \u0027Fake.Describe(int)\u0027 cannot be accessed with an instance reference; qualify it with a type name instead\nFakeTest.cs:99: Member \u0027Fake.Describe(int)\u0027 cannot be accessed with an instance reference; qualify it with a type name instead\nFakeTest.cs:107: Member \u0027Fake.Positive(int)\u0027 cannot be accessed with an instance reference; qualify it with a type name instead\nFakeTest.cs:115: Member \u0027Fake.Positive(int)\u0027 cannot be accessed with an instance reference; qualify it with a type name instead",
  "tests": []
}